config.json

```diff
@@ -270,6 +270,12 @@
       "values": [ true, false ],
       "defaultValue": true
     },
+    "SkipSOS": {
```

Comment:
I do not  think we need this option here.

Reply:
Is there any way to pass MSBuild option to MSBuild through run.cmd? I tries the following changes, but it does not work for the command of the form ``build.cmd release skipnative windowsmscorlib -- /p:SkipA=true``:
```
diff --git a/build.cmd b/build.cmd
index 69597eb00..6ef5f9c19 100644
--- a/build.cmd
+++ b/build.cmd
@@ -520,8 +520,9 @@ if %__BuildCoreLib% EQU 1 (
       set __ExtraBuildArgs=!__ExtraBuildArgs! -EnableProfileGuidedOptimization=true
     )

+    set __ExtraMSBuildArgs=
     if "%__BuildSOS%" == "0" (
-        set __ExtraBuildArgs=!__ExtraBuildArgs! -SkipSOS=true
+        set __ExtraMSBuildArgs=!__ExtraMSBuildArgs! -p:SkipSOS=true
     )

     if /i "%__BuildArch%" == "arm64" (
@@ -540,7 +541,7 @@ if %__BuildCoreLib% EQU 1 (
     set __MsbuildWrn=/flp1:WarningsOnly;LogFile=!__BuildWrn!
     set __MsbuildErr=/flp2:ErrorsOnly;LogFile=!__BuildErr!

-    @call %__ProjectDir%\run.cmd build -Project=%__ProjectDir%\build.proj -MsBuildLog=!__MsbuildLog! -MsBuildWrn=!__MsbuildWrn! -MsBuildErr=!__MsbuildErr! !__nugetBuildArgs! %__RunArgs% !__ExtraBuildArgs! %__UnprocessedBuildArgs%
+    @call %__ProjectDir%\run.cmd build -Project=%__ProjectDir%\build.proj -MsBuildLog=!__MsbuildLog! -MsBuildWrn=!__MsbuildWrn! -MsBuildErr=!__MsbuildErr! !__nugetBuildArgs! %__RunArgs% !__ExtraBuildArgs! %__UnprocessedBuildArgs% -- !__ExtraMSBuildArgs!

     if not !errorlevel! == 0 (
         echo %__MsgPrefix%Error: System.Private.CoreLib build failed. Refer to the build log files for details:
```

Reply:
Oh, ok. I forgot that there is the `run.cmd` layer...